tests/kernel/device/src/main.c

```diff
@@ -78,6 +79,7 @@ void test_dummy_device_pm(void)
 	zassert_false((busy == 0), NULL);
 
 	device_busy_clear(dev);
+	device_set_power_state(dev, DEVICE_PM_ACTIVE_STATE);
```

Comment:
Why do we set to active state here? If it is for coverage improvement, do we not need to test `device_get_power_state` too, after this? I can see `set` is already being used in tests/power, but not `get`.

Reply:
technically yes if the driver is implementing any suspend resume calls. But this is dummy driver which does not have those calls implemented. My intention of this change is to just invoke the device_pm_control_nop() through this call.